orchestrator-salt/src/main/resources/salt/salt/nginx/conf/clouderamanager-ssl-user-facing.conf

```diff
@@ -38,6 +38,8 @@ server {
         proxy_set_header   Upgrade $http_upgrade;
         proxy_set_header   Connection $connection_upgrade;
         proxy_set_header   Referer {{ protocol }}://$host/$1;
+        # Add Security cookie flags
+        proxy_cookie_path ~(.*) "$1; SameSite=none; secure; httponly";
```

Comment:
So cookies MAY have the SameSite set. In that case you should not overwrite it or add it.

Reply:
That is true @biharitomi is there a way to do this? thanks!

Reply:
You can test this change on a simple local env of CB which based on your changes. Every created cluster is going to have this config, so you are able to test this change even in VPC which as proxy to complicate the flow.